src/features/config/form/options.ts

```diff
@@ -12,40 +12,198 @@
 
-import { formMessageDescriptors } from './formatjs-messages'
-import { BIG_NUMBER, IDynamicFieldTypeMapper, TEXT } from './types/types'
-import { MessageDescriptor } from 'react-intl'
-
-export const NATIONAL_ID = 'NATIONAL_ID'
-export const BIRTH_REGISTRATION_NUMBER = 'BIRTH_REGISTRATION_NUMBER'
-export const PASSPORT = 'PASSPORT'
-export const DEATH_REGISTRATION_NUMBER = 'DEATH_REGISTRATION_NUMBER'
-export const DRIVING_LICENSE = 'DRIVING_LICENSE'
-export const REFUGEE_NUMBER = 'REFUGEE_NUMBER'
-export const ALIEN_NUMBER = 'ALIEN_NUMBER'
-export const OTHER = 'OTHER'
-export const NO_ID = 'NO_ID'
-export const SOCIAL_SECURITY_NO = 'SOCIAL_SECURITY_NO'
-
 export const deathDocumentForWhomFhirMapping = {
   DECEASED_ID_PROOF: 'DECEASED_ID_PROOF',
   DECEASED_DEATH_PROOF: 'DECEASED_DEATH_PROOF',
   DECEASED_DEATH_CAUSE_PROOF: 'DECEASED_DEATH_CAUSE_PROOF',
   INFORMANT_ID_PROOF: 'INFORMANT_ID_PROOF'
 }
 
-export const deathDocumentTypeFhirMapping = {
-  HOSPITAL_CERTIFICATE_OF_DEATH: 'HOSPITAL_CERTIFICATE_OF_DEATH',
-  ATTESTED_LETTER_OF_DEATH: 'ATTESTED_LETTER_OF_DEATH',
-  BURIAL_RECEIPT: 'BURIAL_RECEIPT',
-  POLICE_CERTIFICATE_OF_DEATH: 'POLICE_CERTIFICATE_OF_DEATH',
-  MEDICALLY_CERTIFIED_CAUSE_OF_DEATH: 'MEDICALLY_CERTIFIED_CAUSE_OF_DEATH',
-  VERBAL_AUTOPSY_REPORT: 'VERBAL_AUTOPSY_REPORT',
-  CORONERS_REPORT: 'CORONERS_REPORT',
-  BIRTH_CERTIFICATE: 'BIRTH_CERTIFICATE',
-  NATIONAL_ID: 'NATIONAL_ID',
-  PASSPORT: 'PASSPORT',
-  OTHER: 'OTHER'
+/** Mapping of attachment types to identifiers */
+export const attachment = {
+  attestedLetterOfDeath: 'ATTESTED_LETTER_OF_DEATH',
+  birthCertificate: 'BIRTH_CERTIFICATE',
+  burialReceipt: 'BURIAL_RECEIPT',
+  coronersReport: 'CORONERS_REPORT',
+  hospitalCertificateOfDeath: 'HOSPITAL_CERTIFICATE_OF_DEATH',
+  marriageNotice: 'MARRIAGE_NOTICE',
+  medicallyCertifiedCauseOfDeath: 'MEDICALLY_CERTIFIED_CAUSE_OF_DEATH',
+  nationalId: 'NATIONAL_ID',
+  notificationOfBirth: 'NOTIFICATION_OF_BIRTH',
+  other: 'OTHER',
+  passport: 'PASSPORT',
+  policeCertificateOfDeath: 'POLICE_CERTIFICATE_OF_DEATH',
+  proofOfAssignedResponsibility: 'PROOF_OF_ASSIGNED_RESPONSIBILITY',
+  proofOfLegalGuardianship: 'PROOF_OF_LEGAL_GUARDIANSHIP',
+  verbalAutopsyReport: 'VERBAL_AUTOPSY_REPORT'
+}
+
+export const DEATH_ATTACHMENT_TYPES = [
+  attachment.hospitalCertificateOfDeath,
+  attachment.attestedLetterOfDeath,
+  attachment.burialReceipt,
+  attachment.policeCertificateOfDeath,
+  attachment.medicallyCertifiedCauseOfDeath,
+  attachment.verbalAutopsyReport,
+  attachment.coronersReport,
+  attachment.birthCertificate,
+  attachment.nationalId,
+  attachment.passport,
+  attachment.other
+] as const
+
+export const BIRTH_ATTACHMENT_TYPES = [
+  attachment.birthCertificate,
+  attachment.nationalId,
+  attachment.passport,
+  attachment.other,
+  attachment.notificationOfBirth,
+  attachment.proofOfLegalGuardianship,
+  attachment.proofOfAssignedResponsibility
+] as const
+
```

Comment:
Should I move these to `data-generator`? They're not really related to forms but just meta information for data gen